app/bundles/CoreBundle/Assets/css/libraries/libraries.css

```diff
@@ -6371,9 +6371,9 @@ button.close {
   font-weight: normal;
   font-style: normal;
 }
-.fa {
+.fa { 
   display: inline-block;
-  font: normal normal normal 14px/1 FontAwesome;
+  font: normal normal normal 14px/1 FontAwesome,sans-serif;
```

Comment:
It cannot be fixed in this file. libraries.css is compiled when you run the `app/console mautic:assets:generate` which is run before every release. The `.fa` class is defined in the Font Awesome library and we should not hack libraries, because when the library will be updated, the hack will be lost. So what I'd recommend is to add this as a CSS override into `app/bundles/CoreBundle/Assets/css/app/less/custom.less`